qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -1,5 +1,144 @@
-from qcodes.instrument.visa import VisaInstrument
-from qcodes.utils.validators import Numbers
+from qcodes import VisaInstrument, InstrumentChannel
+from qcodes.instrument.parameter import ManualParameter
+from qcodes.utils.validators import Numbers, Bool, Enum, Nothing, Ints
+
+def float_int(val):
+    """
+    Parses int that are returned in exponentiated form (i.e. 1E0)
+    """
+    return int(float(val))
```

Comment:
I wonder if we want to do this as int(round(float(val))) if for some reason float(val) evaluates to just below an int e.g. `9.9` int(val) vil round down to 9

Reply:
In this instance they are all integers, but point taken as a general rule. Changed...